### PR TITLE
fix(popover): Add "active" prop 

### PR DESCRIPTION
**Pull request by pixelflips**
## Description

Adds `active` prop to component to allow to show and hide.

## Screenshots
No visual changes are expected with these updates.

## Testing in `sage-lib`

- Navigate to [popover](http://localhost:4100/?path=/docs/sage-popover--default)
- Verify `active` prop exists and functions as expected.
- Verify popover still functions as expected when interacted with.

## Testing in `kajabi-products`
1. (**LOW**) Adds new active prop to Popover (React).

## Related
https://kajabi.atlassian.net/browse/DSS-501

### CODE REVIEW

File: packages/sage-react/lib/Popover/Popover.jsx [hunk 1]
```diff
@@ -20,7 +21,7 @@ export const Popover = ({
   title,
   ...rest
 }) => {
-  const [selfActive, setSelfActive] = useState(false);
+  const [selfActive, setSelfActive] = useState(active);
```
Comment (ju-Skinner) [inline packages/sage-react/lib/Popover/Popover.jsx:24]: I know you didn't write the initial line of code, but we should change the `selfActive` name to `isActive`. We should follow the same for `setSelfActive`, and rename it to `setIsActive`.

Comment (pixelflips) [reply packages/sage-react/lib/Popover/Popover.jsx]: @ju-Skinner Updated. Great call, that does read much cleaner!

File: packages/sage-react/lib/Popover/Popover.jsx [hunk 2]
```diff
@@ -20,27 +21,27 @@ export const Popover = ({
   title,
   ...rest
 }) => {
-  const [selfActive, setSelfActive] = useState(false);
+  const [isActive, setisActive] = useState(active);
```
Comment (ju-Skinner) [inline packages/sage-react/lib/Popover/Popover.jsx:24]: `setisActive` should be camel-cased `setIsActive`.

Comment (pixelflips) [reply packages/sage-react/lib/Popover/Popover.jsx:24]: doh! all lowercase is a hard habit to break. Should be good now, thanks for catching it!

**Review by ju-Skinner — APPROVED** [review]
Great job. Thanks for taking this on.

**Pull request merged**